test/verify/check-storage-mounting

```diff
@@ -314,7 +314,13 @@ ExecStart=/usr/bin/sleep infinity
         self.dialog_wait_open()
         self.dialog_set_val("mount_options.extra", "hurr")
         self.dialog_apply()
-        self.dialog_wait_alert("bad option")
+        try:
+            # Modern versions of mount give a specific error
+            self.dialog_wait_alert("Unknown parameter 'hurr'")
```

Comment:
I quickly checked the code and dialog_wait_alert() should only wait for 15s, not 60, right? The latter would start to be rather painful.

Reply:
Yeah. We might switch the test around, or do something more elaborate...

Reply:
I did that, no more waiting for a timeout.